app/controllers/org_admin/sections_controller.rb

```diff
@@ -36,7 +36,8 @@ def show
     def edit
       section = Section.includes({phase: :template}, questions: [:question_options, { annotations: :org }]).find(params[:id])
       authorize section
-      render partial: 'edit', 
+      # User cannot edit a section if its not modifiable or the template is not the latest redirect to show
+      render partial: (section.modifiable? && section.phase.template.latest? ? 'edit' : 'show'), 
```

Comment:
we should add a test to inspect the DOM and check whether or not is displayed the edit/show partial depending on the case.